Dockerfile

```diff
@@ -17,16 +17,17 @@ WORKDIR /mnt/workdir
 # from the mounted directory. But that's not available during the
 # docker build, so we need to copy them in to pre-install the Gems
 
-COPY Gemfile ./Gemfile
-COPY Gemfile.lock ./Gemfile.lock
+COPY Gemfile Gemfile.lock ./
 
 # Gems will be installed under GEM_HOME which is set by the ruby image.
 # See https://hub.docker.com/_/ruby for details.
 
-RUN gem update --system && bundle install && gem cleanup
+RUN gem update --system \
+  && bundle install \
+  && gem cleanup
 
-ENV HOST=0.0.0.0
-ENV PORT=4000
+ENV HOST=0.0.0.0 \
+    PORT=4000
```

Comment:
I think it's better as two separate `ENV` lines. They are two independent environment declarations. This new syntax makes it seem like the two are related in some way, and might be confused with something like `ENV PROPWITHEMBEDDEDSPACE=0.0.0.0\ 4000`

Reply:
Reverted in 2a11117